include/engine/routing_algorithms/routing_base_mld.hpp

```diff
@@ -73,8 +74,8 @@ inline bool checkParentCellRestriction(CellID cell, LevelID, CellID parent)
 using PackedEdge = std::tuple</*from*/ NodeID, /*to*/ NodeID, /*from_clique_arc*/ bool>;
 using PackedPath = std::vector<PackedEdge>;
 
-template <bool DIRECTION, typename OutIter>
-inline void retrievePackedPathFromSingleHeap(const SearchEngineData<Algorithm>::QueryHeap &heap,
+template <bool DIRECTION, typename Algorithm, typename OutIter>
+inline void retrievePackedPathFromSingleHeap(const typename SearchEngineData<Algorithm>::QueryHeap &heap,
```

Comment:
Yes. But also, we need to skip include tss with that way. I thought that SearchEngineData template specialization will be better ...